test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test user accounts for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API will return synthetic VA records for all test accounts.
+
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+### Logging in with ID.me
+
+To log in to the sandbox environment using ID.me, make sure you:
+
+1. Choose an account that is specific to the scenario you want to test.
+2. Select **ID.me** to sign in to the sandbox environment.
```

Comment:
We've already started a thread about this in Slack, but to document it here:

I think we should include a link to the page where the user is supposed to enter the usernames and passwords and explain how to find the token once they've entered them. I wasn't able to find this information on my own.

Reply:
Not sure I'm following along, but would this issue be solved by public platform's plan to send users to a hidden page on the public portal that has the credentials?  If not, we may want to let them know that there's a piece missing for how to login / find the token.

Reply:
I think we can hold off on addressing this for now - knowing that the user will be interacting with the public portal, I think that's the best place to make the information I talked about available (not here)